superagi/tools/webscaper/web_scraper_toolkit.py

```diff
@@ -5,7 +5,7 @@
 
 
 class WebScrapperToolkit(BaseToolkit, ABC):
-    name: str = "Web Scrapper Toolkit"
+    name: str = "Web Scraper Toolkit"
     description: str = "Web Scrapper tool kit is used to scrape web"
```

Comment:
Description also needs to be updated. Change the spelling in utils.js also, so that the icon doesn't get affected. Test it once after the changes are made